desktop/release-notes.md

```diff
@@ -24,35 +24,97 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
+- [Docker Scout CLI](../scout/index.md#docker-scout-cli) now displays base image update and remediation recommendations using `docker scout recommendations`. It also displays a short overview of an image using `docker scout quickview` commands.
+- You can now search for extensions direct from the Marketplace, as well as using **Global Search**.
+- Fixed a bug where `docker buildx` container builders would lose access to the network after 24hrs.
+- Reduced how often users are prompted for feedback on Docker Desktop.
+- Removed minimum VM swap size.
+- Added support for subdomain match, CIDR match, `.` and `_.` in HTTP proxy exclude lists.
+- Fixed a bug in the transparent TLS proxy when the Server Name Indication field is not set.
+- Fixed a grammatical error in Docker Desktop engine status message.
+
+### For Windows
+
+- Fixed a bug where `docker run --gpus=all` hangs. Fixes [docker/for-win#13333](https://github.com/docker/for-win/issues/13333).
+- Fixed a bug where Registry Access Management policy updates were not downloaded.
+- Docker Desktop now allows Windows containers to work when BitLocker is enabled on `C:`.
+- Docker Desktop with the WSL backend no longer requires the `com.docker.service` privileged service to run permanently. For more information see [Permission requirements for Windows](https://docs.docker.com/desktop/windows/permission-requirements/).
+
+### For Mac
+
+- Fixed a performance issue where attributes stored on the host would not be cached for VirtioFS users.
+- The first time Docker Desktop for Mac is launched, the user is presented with an installation window to confirm or adjust the configuration that requires privileged access. For more information see [Permission requirements for Mac](https://docs.docker.com/desktop/mac/permission-requirements/).
+- Added the **Advanced** tab in **Settings**, where users can adjust the settings which require privileged access.
+
+### For Linux
+
+- Fixed a bug where the VM networking crashes after 24h. [docker/for-linux#131](https://github.com/docker/desktop-linux/issues/131)
+
+### Security
+
+#### For all platforms
+
+- Fixed a security issue with the Artifactory Integration where it would fall back to sending registry credentials over plain HTTP if HTTPS check failed. Only users who have `Access experimental features` enabled are affected. Fixes [docker/for-win#13344](https://github.com/docker/for-win/issues/13344).
+
+#### For Mac
+
+- Removed the `com.apple.security.cs.allow-dyld-environment-variables` and `com.apple.security.cs.disable-library-validation` entitlements which allow an arbitrary dynamic library to be loaded with Docker Desktop via the `DYLD_INSERT_LIBRARIES` environment variable.
+
+### Known Issues
+
+- **Dev environments** will not work properly when configuring the installation of Docker's CLI tools as User.
```

Comment:
Add 
#### For Mac
or 
**Dev environments** will not work properly on Mac when....

Reply:
I took the second option!